### PR TITLE
AI-Training-Best-Practices

### PR DESCRIPTION
**Pull request by siarhei-leuchanka**
Adding a page dedicated to AI training best practices. The first chapter is about queues and when to consider splitting them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,11 @@
     "@docusaurus/preset-classic": "^3.4.0",
     "@mdx-js/react": "^3.0.1",
     "clsx": "^2.1.1",
+    "d3": "^7.9.0",
     "prism-react-renderer": "^2.3.1",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "venn.js": "^0.2.20"
```
Comment (mrtnzlml) [inline package.json:25]: @siarhei-leuchanka Well done; I see you went above and beyond with this! 👍 

I am a bit hesitant to accept this change, though, as this dependency [doesn't seem to be very maintained](https://github.com/benfred/venn.js). Additionally, I am trying to keep the maintenance cost of this project to a minimum, and I wouldn't like to maintain custom React components **if not needed**.

Have you considered using Draw.io, as we are doing it in other places? For example: https://github.com/rossumai/university/tree/aaf5f0343db2853e6fec98fe76c854710b7fdb73/docs/learn/sandboxes/_img

File: package-lock.json
Comment (mrtnzlml) [inline package-lock.json:1]: Please, consider removing this file. We use `yarn.lock` instead.

File: package.json (hunk 1 above)
Comment (siarhei-leuchanka) [reply package.json:25]: @mrtnzlml sure, did not spot it. I will redo it with draw.io

**Comment by siarhei-leuchanka** [comment]
I removed the Venn diagram for now. This was optional.

**Pull request merged**